scripts/deploy.sh

```diff
@@ -39,6 +38,14 @@ if [[ ${TRAVIS_TAG} =~ ^[0-9]+(\.[0-9]+){2}$ ]]; then
     GCR_VERSION=${TRAVIS_TAG}
     GCS_PROJECT="eha-data"
     GCR_PROJECT="production-228613"
+    RELEASE_BUCKET="aether-releases"
```

Comment:
this one too

Reply:
what would I do without you 💯

Reply:
I meant, `export RELEASE_BUCKET="aether-releases"`